git-hg/updatetags8.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash -x
+#
+# ===========================================================================
+# (c) Copyright IBM Corp. 2018 All Rights Reserved
+# ===========================================================================
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
+#
+# ===========================================================================
+
+set -euo pipefail
+
+echo "Common defs"
+
+# shellcheck disable=SC1091
+modules=(corba langtools jaxp jaxws nashorn jdk hotspot)
+
+# These variables needs to define in Jenkin's job pipeline
+#WORKSPACE="<Define local workspace path where clone and merge is going to take place>"
+#GITHUB_PROJECT="<Path for extension git repo>"
+#GITHUB_REPO="<extention repo name, where merge will be pushed>" 
+GITHUB_PROJECT=git@github.com:AdoptOpenJDK
+GITHUB_REPO=openjdk-jdk8u
+
+rm -rf $WORKSPACE/$GITHUB_REPO
+rm -rf $WORKSPACE/openjdk
+
+mkdir $WORKSPACE/$GITHUB_REPO
+mkdir $WORKSPACE/openjdk
+mkdir $WORKSPACE/openjdk/mirror
+
+cd $WORKSPACE/$GITHUB_REPO
+
+echo NOTE: This script requires git 2.16.2 or later and git-remote-hg from
+echo https://raw.githubusercontent.com/felipec/git-remote-hg/master/git-remote-hg
+git --version
+
+# Clone current repo
+echo "Clone current $GITHUB_REPO"
+git clone $GITHUB_PROJECT/$GITHUB_REPO.git
+cd $GITHUB_REPO
+#git fetch --tags
+#OLDTAG=$(git describe --abbrev=0 --tags) 
+OLDTAG=none
+echo "Current openjdk level is $OLDTAG"
+
+# Clone current openjdk
+echo "Get base openjdk repository" 
+cd $WORKSPACE/openjdk/mirror
+git init
+git clone --bare hg::http://hg.openjdk.java.net/jdk8u/jdk8u
+
+cd jdk8u.git
+
+git filter-branch -f --index-filter 'git rm -r -f -q --cached --ignore-unmatch .hg .hgignore .hgtags README get_source.sh' --prune-empty --tag-name-filter cat -- --all
```

Comment:
probably don't want to filter out the `README` here.

Reply:
I suspect that was from the openj9 team replacing it with their own ... Although it's contents (http://hg.openjdk.java.net/jdk8u/jdk8u/file/0c4c976612d3/README) is all about making sure you have all the repositories which may just confuse people ... But then we leave the equivalent doc in for other releases ... OK I'll remove it for now (although the current repo doesn't have one so I'll add it back in)